Autoload/Network.gd

```diff
@@ -0,0 +1,50 @@
+extends Node
+
+var port = 1024
+var max_clients = 3
+
+var server = null
+var client = null
+
+var ip_address = ""
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	for ip in IP.get_local_addresses():
+		if ip.begins_with("192.168.") or ip.begins_with("10.") or ip.begins_with("172.16") and not ip.ends_with(".1"):
+			ip_address = ip
+	
+	get_tree().connect("connected_to_server", self, "_connected_to_server")
```

Comment:
I see these magic string multiple times now... should move to a string constant.

Reply:
Are you referring to the connect method of the scene tree? If so, then I should explain that those are used to connect to a sort of signal that calls the function in a specific script. The syntax being:
`get_tree().connect("signal", object, "function")`

Reply:
k, when I see strings like `"connected_to_server"` repeated it makes me nervous